### DISCUSSION TITLE
Split the editor features among different screens

### DISCUSSION THREAD

**Discussion by pfspace** [godotengine/godot-proposals#4043] [Editor]
I think it would be nice if we could split the editor features among different screens as we can do in the main editor as docks. I don't know how complex that would be, but if at launch Godot draw canvas for each available screen, users would be able to adjust  the main editor to their needs treating those canvas as docks. The script editor could be displayed in one screen and the nodes editor in another, for example.

This would prevent loss of time caused by switching from 2D/3D scenes editor to Script editor often.

Thank you.

**Comment by Calinou** [comment]
This is already being tracked in a proposal: https://github.com/godotengine/godot-proposals/issues/28

**Reply by pfspace** [reply]
I'm sorry. I didn't see that one.
Thank you.
